### ISSUE TITLE
Glutathione peroxidase low in erythrocytes

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11147]
**Preferred term label:**
Glutathione peroxidase low in erythrocytes

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Glutathione peroxidase deficiency

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001314
- label: Reduced erythrocyte glutathione peroxidase
- parents: [Abnormal erythrocyte enzyme concentration or activity (HP:0030272)](https://hpo.jax.org/app/browse/term/HP:0030272)
- pmid: 
- def: Activity or concentration of in the level of glutathione peroxidase (EC 1.11.1.9) in erythrocytes below the lower limit of normal.
- comment: Glutathione peroxidase catalzyes the reaction 2 glutathione + H2O2 = glutathione disulfide + 2 H2O.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11147
- synonyms: Glutathione peroxidase low in erythrocytes(NONE)
